release/build_nightly.sh

```diff
@@ -7,6 +8,13 @@ function main {
 
 		LIFERAY_COMMON_DOWNLOAD_SKIP_CACHE="true" LIFERAY_RELEASE_GIT_REF="master" LIFERAY_RELEASE_OUTPUT="nightly" LIFERAY_RELEASE_UPLOAD="true" ./build_release.sh
 
+		if [ "${?}" -ne 0 ]
```

Comment:
@kiwm, can you please replace it with `if [ "${?}" -ne 0 ] && [ "${1}" == "--no-sleep" ]`?

This way we won't exit nightly execution if this is being executed on BChan's end.